modules/secrets-store-aws-secrets-manager.adoc

```diff
@@ -0,0 +1,357 @@
+----
++
+[NOTE]
+====
+This configuration is modified from the configuration provided in the link:https://github.com/aws/secrets-store-csi-driver-provider-aws#installing-the-aws-provider[AWS documentation] so that it works properly with {product-title}.
+====
+
+.. Grant privileged access to the `csi-secrets-store-provider-aws` service account by running the following command:
++
+[source,terminal]
+----
+$ oc adm policy add-scc-to-user privileged -z csi-secrets-store-provider-aws -n openshift-cluster-csi-drivers
+----
+
+.. Create the provider resources by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f aws-provider.yaml
+----
+
+. Grant permission to allow the service account to read the AWS secret object:
+
+.. Create a directory to contain the credentials request by running the following command:
++
+[source,terminal]
+----
+$ mkdir credentialsrequest-dir-aws
+----
+
+.. Create a YAML file with the following configuration for the credentials request:
++
+.Example `credentialsrequest.yaml` file
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: aws-provider-test
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries:
+    - action:
+      - "secretsmanager:GetSecretValue"
+      - "secretsmanager:DescribeSecret"
+      effect: Allow
+      resource: "arn:*:secretsmanager:*:*:secret:testSecret-??????"
+  secretRef:
+    name: aws-creds
+    namespace: test-namespace
+  serviceAccountNames:
+  - aws-provider
+----
++
+// TODO `resource` value: should we say something about this? I changed it from "secret:xxiaSecret" to "secret:testSecret". What are the "??????" supposed to be?
+
+.. Retrieve the OIDC provider by running the following command:
++
+[source,terminal]
+----
+$ oc get --raw=/.well-known/openid-configuration | jq -r '.issuer'
+----
++
+// TODO: Can this be rewritten to use jsonpath or something else instead of jq? (To avoid making users install jq)
++
+.Example output
+[source,terminal]
+----
+https://<name>-oidc.s3.<aws_region>.amazonaws.com
+----
+Copy the `<oidc_provider_arn>` from the output to use in the next step. For example, `<name>-oidc.s3.<aws_region>.amazonaws.com`.
+
+.. Use the `ccoctl` tool to process the credentials request by running the following command:
++
+[source,terminal]
+----
+$ ccoctl aws create-iam-roles \
+    --name test-role --region=<aws_region> \
+    --credentials-requests-dir=credentialsrequest-dir-aws \
+    --identity-provider-arn <oidc_provider_arn> --output-dir=credrequests-ccoctl-output
+----
++
+.Example output
+[source,terminal]
+----
+2023/05/15 18:10:34 Role arn:aws:iam::<aws_account_id>:role/test-role-test-namespace-aws-creds created
+2023/05/15 18:10:34 Saved credentials configuration to: credrequests-ccoctl-output/manifests/cert-manager-aws-creds-credentials.yaml
```

Comment:
No `cert-manager`. You can hide line 218 with just ellipsis `...` given line 218 does not contain info used later.
```suggestion
...
```

Reply:
Good catch, copy paste from cert-manager file and didn't replace this one properly